src/main/java/org/schemaspy/connection/SemicolonSeparated.java

```diff
@@ -0,0 +1,39 @@
+package org.schemaspy.connection;
+
+import java.util.Properties;
+import java.util.StringTokenizer;
+
+/**
+ * Specifies connection properties to use in the format:
+ * <code>key1\=value1;key2\=value2</code><br>
+ * user (from -u) and password (from -p) will be passed in the
+ * connection properties if specified.<p>
+ * This is an alternative form of passing connection properties than by file
+ */
+public class SemicolonSeparated {
```

Comment:
I have somewhat of a feeling that if `;` was replaced with `\n` and the `\\=` was just a regular `=` we could skip our processing and just send it straight to one of the load functions of properties.

Reply:
Happy to try that when this is merged. Or do you want to?

Reply:
It can be done at a later stage, it will be an experiment to see if it possible.